README.md

```diff
@@ -14,18 +14,95 @@
 
 </div>
 
-Vello is a 2d graphics rendering engine, using [`wgpu`].
+Vello is an experimental 2d graphics rendering engine written in Rust, using [`wgpu`].
 It efficiently draws large 2d scenes with interactive or near-interactive performance.
 
-<!-- Impressive picture here -->
-
-It is used as the rendering backend for [Xilem], a UI toolkit.
-
 Quickstart to run an example program:
 ```shell
 cargo run -p with_winit
 ```
 
+![image](https://github.com/linebender/vello/assets/8573618/cc2b742e-2135-4b70-8051-c49aeddb5d19)
+
+It is used as the rendering backend for [Xilem], a native Rust GUI toolkit.
+
+
+## Motivation
+
+Vello is meant to fill the same place in the graphics stack as other vector graphics renderers like [Skia](https://skia.org/), [Cairo](https://www.cairographics.org/), and its predecessor project [Piet](https://www.cairographics.org/).
+On a basic level, that means it provides tools to render shapes, images, gradients, texts, etc, using a PostScript-inspired API, the same that powers SVG files and [the browser `<canvas>` element](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D).
+
+Vello's selling point is that it gets better performance than other renderers by better leveraging the GPU.
+In traditional PostScript renderers, some steps of the render process like sorting and clipping either need to be handled in the CPU or done through the use of intermediary textures.
+Vello avoids this by using prefix-scan algorithms to parallelize work that usually needs to happen in sequence, so that work can be offloaded to the GPU with minimal use of temporary buffers.
+
+
+## Getting started
+
+Vello is meant to be integrated deep in UI render stacks.
+While drawing in a Vello scene is easy, actually rendering that scene to a surface setting up a wgpu context, which is a non-trivial task.
+
+To use Vello as the renderer for your PDF reader / GUI toolkit / etc, your code will have to look roughly like this:
+
+```rust
+// Initialize wgpu and get handles
+let device: wgpu::Device = ...;
+let queue: wgpu::Queue = ...;
+let render_surface: wpg::RenderSurface<'_> = ...;
```

Comment:
What is `wpg::RenderSurface`? I haven't been able to find any matching types. wgpu's [`Surface`](https://docs.rs/wgpu/latest/wgpu/struct.Surface.html) doesn't have that prefix